app/assets/stylesheets/custom.scss

```diff
@@ -441,9 +462,104 @@ ul.edit-listings  {
 //    =========================*/
 
 .listing-create-form {
-  background: #c5eaea;
+  background: rgba(131, 194, 177, 0.6);
 }
 
 .edit-new-listing {
   margin-top: 70px;
 }
+
+#nightly_rate {
+  margin-top: 40px;
+}
+
+select#_max_guests.form-control {
```

Comment:
Avoid using id selectors<br>Avoid qualifying id selectors with an element.<br>Avoid qualifying class selectors with an element.<br>Selector `_max_guests` should be written in lowercase with hyphens